microovn/src/charm.py

```diff
@@ -11,11 +15,64 @@ def get_hostname():
 def mirror_id(hostname):
     return MIRROR_PREFIX + hostname
 
+def call_microovn_command(*args):
+    result = subprocess.run(
```

Comment:
The `subprocess` package does not appear to be imported (possibly being the source of the CI test failure?).